.github/workflows/release-reference-antora-extension.yaml

```
@@ -26,7 +26,7 @@ name: Release reference-antora-extension
 
 
 jobs:
-  release-antora-extension:
+  release-reference-antora-extension:
```

```suggestion
  release-antora-extensions:
```

We could keep the same workflow and use matrix strategy to run it twice in two different directories, keeps the maintenance overhead low. See an example [here](https://github.com/hacbs-contract/ec-policies/blob/398a79a4830ee89d917aa1ad6f5a3fb78e238f3b/.github/workflows/docs.yaml#L12-L18)

---

I'll check into that.